docs/instruction_for_testing.md

```diff
@@ -0,0 +1,135 @@
+  --password Myp@SS0rdsS1! \
+  --files-folder test
+```
+
+Where `url`, `account`, `login`, `password` detail the Conjur server-specific information you previously deployed. The type Conjur server deployed impacts where certain tests will pass or fail. For example, if you have configured a Conjur server with a verifiable CA-signed certificate, then `test_https_cli_fails_if_cert_is_bad` and `test_https_cli_fails_if_cert_is_not_provided` should fail. "Why" you may ask? See the developer note above each test or see the issue created [here](https://github.com/cyberark/conjur-api-python3/issues/209). For all other flows, these tests should pass.
+
+Congrats! You ran project's integration tests.
+
+## Running tests with configurable environment
+
+As part of our test suite, we also want to test the CLI against different types of Conjur server environments. We want to validate the flow when the CLI makes a request to a server and the certificate configured is either 1. a known, verifiable CA-signed certificate or 2. a non-verifiable CA-signed certificate.
+
+To accomplish the first flow, at a high level, you will need to:
+
+1. Configure the certificate on Conjur server
+1. Add CA certificate to store
+1. Run the `integrations_tests_runner`
+
+#### Configure the certificate on Conjur server
+
+On your Conjur server, you will need to create and configure the certificate on the Conjur server by following this guide (TODO ADD LINK).
+
+#### Add CA certificate to store
+
+Python comes with a list of trusted CAs. Every certificate signed by those CAs will be considered trusted and verifiable. Those CAs are located in a file called `cacerts.pem`. To manipulate the types of CAs the executable trusts, you will need to add the CA pem to that `cacerts.pem` and rebuild the executable. This is exactly what we will do!
+
+1. Choose a platform to run the additional test flow (Hint: macOS should be the easiest). 
+1. Locate the path to this file and run `python3 -c "import certifi; print(certifi.where())"`. 
+1. Once you have the path, edit the `cacerts.pem` file.
+1. Add the CA pem to the top of the file. Adding this CA certificate to the system certificate store allows the certificate to be verifiable.
+
+#### Run the `integrations_tests_runner`
+
+Once you have added the CA pem to the `cacerts.pem`, you will need to rebuild the `integrations_tests_runner`. 
+
+1. Remove the cached directories `build` and `dist`.
+1. Run `pyinstaller -F test/util/test_runners/integrations_tests_runner.py`. 
+
+Note that for this flow, two tests mentioned in the previous section *should fail*.
+
+For the non-verifiable CA-signed certificate flow, you will need to run through the flow described in this section again. All you need to do is:
+
+1. Remove the cached directories.
+1. Remove the CA pem from the `cacerts.pem`.
+1. Rebuild the `integrations_tests_runner`. 
+
+Note that for this flow, two tests mentioned in the previous section *should pass*.
+
+## Miscellaneous tests
```

Comment:
The following tests should be run before release only (enough time to fix new issues)

Reply:
Done